Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -384,13 +384,35 @@ extension SwiftLanguageServer {
           }
           let isInsertTextSnippet = clientSupportsSnippets && text != insertText
 
+          let textEdit: TextEdit?
+          if let text = text {
+            let bytesToErase: Int = value[self.keys.num_bytes_to_erase] ?? 0
```

Comment:
Theoretically this needs to be converted from UTF-8 byte count to UTF-16 code unit count.  In practice I think we only currently delete ASCII so it might not be noticeable.  At the very least it deserves a FIXME.